### DISCUSSION TITLE
Réaction secrète

### DISCUSSION THREAD

**Discussion by KirIcare** [Crownicles/Crownicles#2381] [Nouvelles idées]
Comme le dit le titre, faire une réaction secrète. Je m'explique. Je pensais faire un réaction qui apparaîtrait de manière totalement random (avec un très faible taux d'apparition) et qui serais commune à tous les évent, indépendamment du lieu. Concrètement c'est le même principe que l'attaque de guilde lors des combats de boss sur l'île sauf que là, c'est une réaction (pourquoi pas l'émoji :fog:) qui apparaîtrait comme une réaction classique d'un évent, un choix supplémentaire quoi. Ça serais une issue bénéfique, impossible à optimisée. Simplement de la luck. Évidemment si on le fait, faudrait que le lore de cette issue colle à tous les lieux genre "une entité mystérieuse qui nous atteint à la fin de notre trajet" ou "une brume soudaine qui nous enveloppe et quand elle se dissipe on se retrouve dans un endroit inconnu, hors du temps".
Par contre j'ai 0 idées sur les récompenses d'une telle issue, faut d'abord voir si le concept est faisable xD

**Comment by Sousoulman** [comment]
Je suis d'accord sur l'idée mais peut être plus un endroit spécial au moment du choix de la destination genre pour allez au château ou dans une zone secrète

**Reply by KirIcare** [reply]
Ça perds un peu de son intérêt non ? Le but c'est que ce soit 100% random, si c'est un endroit spécial, suffit de boucler avec cet endroit jusqu'à tomber sur la réaction...

**Comment by leuhvraiminoryo** [comment]
Cela me semble intéressant, mais bien difficile à justifier par le lore (remember, on a pas le droit à du fantasy, et ce que tu as proposé en exemple tombe dans cette catégorie).

En réalité, je pense que remanier ça pour faire que chaque évent a sa propre "réaction secrète" avec une faible chance de venir, et qui représente un choix de plus pertinent avec la situation de l'évent, donnerait quelque chose d'encore mieux :)

**Reply by KirIcare** [reply]
Peut être. C'est l'intérêt des suggestions, recevoir les avis des autres pour les améliorer x). Je suis d'accord avec ton idée, ça collerais encore avec mon idée de base, juste ça rajoute bcp plus de travail xD

**Comment by Sousoulman** [comment]
Non justement une sortie que tu peux avoir de n'importe où genre "un convoi est venu vous chercher car le roi veut vous parler" et ou que l'on soit on peut avoir cette possibilité (si elle apparait) avec par exemple 1h de trajet et un évènement du style un combat pve très dur

**Reply by leuhvraiminoryo** [reply]
sauf que ça colle alors plus à juste... un nouveau mini-évent
